src/main/java/networkbook/logic/commands/EditCommand.java

```diff
@@ -147,8 +147,8 @@ public String toString() {
      * corresponding field value of the person.
      */
     public static class EditPersonDescriptor {
-        private Name name;
-        private Phone phone;
+        private Name name = null;
+        private Optional<UniqueList<Phone>> phones = Optional.empty();
```

Comment:
This is okay for now, but to keep up with recommended practice of using `Optional`, please refer to my comment [here](https://github.com/AY2324S1-CS2103T-T08-2/tp/pull/88#discussion_r1362397800) and change the implementation when you are free, thanks!